contracts/src/v0.8/automation/interfaces/zksync/IGasBoundCaller.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.19;
+
+address constant GAS_BOUND_CALLER = address(0xc706EC7dfA5D4Dc87f29f859094165E8290530f5);
```

Comment:
I guess this contract is about [this](https://docs.zksync.io/zk-stack/components/smart-contracts/system-contracts#gasboundcaller). which network is this address deployed to?